### ISSUE TITLE
Extended enum values are not recognized 

### ISSUE THREAD

**Issue by DanielOrtel** [jimkyndemeyer/js-graphql-intellij-plugin#293]
I really appreciate this plugin and I'm not trying to be a bother with opening issues left and right, but I did notice another thing about enums. 

## Version and Environment Details
__Operation system__: 
macOS Mojave 10.14.6

__IDE name and version__: 
Webstorm 2019.2.3

__Plugin version__: 
2.2.0

## Expected Behaviour
When extending an enum type by `extend enum MyEnum {EXTRA_VALUE}`, the plugin should correctly recognize `EXTRA_VALUE` as a value of `MyEnum`.

## Actual Behaviour
The plugin displays an error for every value which is not present in the original definition of the enum. 

<img width="1147" alt="Screen Shot 2019-10-16 at 6 53 23 PM" src="https://user-images.githubusercontent.com/19431728/66936824-00e71800-f047-11e9-8c9f-834c2993c509.png">

## Steps to Reproduce / Link to Repo with Reproduction and Instructions
```javascript
type User {
  id: ID
  email: String
}

enum AuthRoles {
  USER
  CREW
  ADMIN
}

extend enum AuthRoles {
  AUTHENTICATED
  CURRENT_USER
  OR
  AND
  NOT
}

type Query {
  getUser(id: ID!): User @auth(if: [ADMIN OR CREW OR CURRENT_USER])
}
```

**Comment by BrentFarese** [comment]
I am seeing a related issue when attempting to extend an already-defined interface as contemplated by the [GraphQL spec](http://spec.graphql.org/draft/#sec-Interfaces.Interfaces-Implementing-Interfaces).

**Comment by jimkyndemeyer** [comment]
Extended enum values and interfaces implementing interfaces should be supported in the upcoming 2.5.0, currently pending JetBrains approval.

**Comment by jimkyndemeyer** [comment]
Ah, jumped the gun. The "interface implements" change is in 2.5.0, but it still appears that graphql-java 15.0 doesn't pick up on enum extensions when it's used inside a directive.

**Comment by clementohNZ** [comment]
+1

**Comment by vepanimas** [comment]
Fixed in 3.0.0.
